kiali/docinfo.xml

```diff
@@ -0,0 +1,12 @@
+<title>Kiali Operator provided by Red Hat</title>
+<productname>{product-title}</productname>
+<productnumber>{product-version}</productnumber>
+<subtitle>Using Kiali Operator provided by Red Hat</subtitle>
+<abstract>
+    <para>This documentation provides information about the Kiali Operator provided by Red Hat, and how to view the data that flows through your application.
```

Comment:
You wouldn't do this above, but for this paragraph you would add {nbsp} (non-breaking spaces) between "Red" and "Hat": https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces

```suggestion
    <para>This documentation provides information about the Kiali Operator provided by Red{nbsp}Hat, and how to view the data that flows through your application.
```

Though ensure that this does not render in a preview. I also wouldn't made this change in such places as the orgname below, since that's pretty standardized across out docs.

Reply:
used attribute